pkg/ingester/index/index.go

```diff
@@ -100,10 +100,11 @@ func (shard *indexShard) add(metric []client.LabelPair, fp model.Fingerprint) {
 	defer shard.mtx.Unlock()
 
 	for _, pair := range metric {
-		name, value := model.LabelName(pair.Name), model.LabelValue(pair.Value)
-		values, ok := shard.idx[name]
+		value := model.LabelValue(pair.Value)
+		values, ok := shard.idx[model.LabelName(pair.Name)]
```

Comment:
These 2 lines (4 changes) are a no-op, right?

Reply:
There's no functional change but one fewer allocation the new way.